pdm/cli/commands/config.py

```diff
@@ -43,6 +43,11 @@ def _list_config(self, project: Project, options: argparse.Namespace) -> None:
         ui.echo("Home configuration ({}):".format(project.global_config._config_file))
         with ui.indent("  "):
             for key in sorted(project.global_config):
+                try:
+                    project.global_config._config_map[key]
```

Comment:
Why not assign to an intermediate variable as it is used later.

Reply:
Updated